strawberry_django_plus/filters.py

```diff
@@ -1,6 +1,7 @@
 from typing import Any, Callable, Optional, Sequence, Type, TypeVar, cast
 
 from django.db.models.base import Model
+from django.db.models.sql.query import get_field_names_from_opts
```

Comment:
This seems to be failing due to not being an export in the `__all__` despite being a valid import, even in Django v4